back/app/services/settings_service.rb

```diff
@@ -14,7 +14,10 @@ def add_missing_features(settings, schema)
     else
       res = settings.clone
       missing_features.each do |f|
-        res[f] = { 'allowed' => false, 'enabled' => false }
+        res[f] = {
+          'allowed' => (default = default_setting(schema, f, 'allowed')).nil? ? false : default,
+          'enabled' => (default = default_setting(schema, f, 'enabled')).nil? ? false : default
```

Comment:
```suggestion
          'allowed' => !!default_setting(schema, f, 'allowed'),
          'enabled' => !!default_setting(schema, f, 'enabled')
```

Reply:
Good point. Added it in the base branch 18a2b84f8a75390e9e06df8e82c501e0056fe774